### PR TITLE
TRI-750/add-documentation-for-failed-statuses

### PR DESCRIPTION
**Pull request by krisha-sc**
## Ticket Link
[TRI-750](https://boltlogistics.atlassian.net/browse/TRI-750)

## Tech Notes
- update tracking response to include all order statuses (including failed)
- update tracking response to include `verificationPhotosUrls` and `signaturePhotosUrls`

### CODE REVIEW

**Review by alexcheng068 — APPROVED** [review]

File: swagger.yml [hunk 1]
```diff
@@ -852,40 +852,52 @@ definitions:
...
-              "eventDescription": "The shipment has been delivered.",
-              "timestamp": 1660052210000,
-              "eventName": "SHIPMENT_DELIVERED",
+			        "eventName": "FAILED",
+			        "timestamp": 1668461456000,
+			        "eventDescription": "customer_refused"
+		        },
+            {
+              "eventName": "OUT_FOR_DELIVERY",
+              "timestamp": 1668462845000
```
Comment (snoopy55619819) [inline swagger.yml]: Is it just me or is the spacing here weird?

**Review by snoopy55619819 — APPROVED** [review]
Nice!

**Review by mhydedesousa — APPROVED** [review]
LGTM

File: swagger.yml (hunk 1 above)
Comment (krisha-sc) [reply swagger.yml]: @snoopy55619819 thats weird, it doesn't look like that on my local. Going to fix it on the editor here

**Pull request merged**